install_config/storage_examples/gluster_backed_registry.adoc

```diff
@@ -119,33 +134,38 @@ server=/17.30.172.in-addr.arpa/127.0.0.1#8053
 ----
 ====
 
-With these settings, `dnsmasq` will pull from the `/etc/hosts` file on the master node.
-Add the appropriate hostnames and IPs for all necessary hosts.
+With these settings, *dnsmasq* will pull from the *_/etc/hosts_* file on the
+master node.
+
+Add the appropriate host names and IPs for all necessary hosts.
 
-In *master-config.yaml*, change `bindAddress` to:
+In *_master-config.yaml_*, change `*bindAddress*` to:
 ====
 ----
 dnsConfig:
  bindAddress: 127.0.0.1:8053
 ----
 ====
 
-When pods are created, they receive a copy of */etc/resolve.conf*, which typically contains
-only the master DNS server so they can resolve external DNS requests.  To enable internal
-DNS resolution, insert the dnsmasq server to the beginning.  This way, dnsmasq will attempt to resolve
-requests internally first.
+When pods are created, they receive a copy of *_/etc/resolve.conf_*, which
+typically contains only the master DNS server so they can resolve external DNS
+requests. To enable internal DNS resolution, insert the *dnsmasq* server to the
+beginning. This way, *dnsmasq* will attempt to resolve requests internally
```

Comment:
"To enable internal DNS resolution, insert the dnsmasq server to the beginning."

I'm guessing this is like any other resolv.conf file thing. Maybe:

"To enable internal DNS resolution, insert the dnsmasq server information at the top of the server list." or something.